src/components/useCurrencyConverter.js

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect } from "react";
+
+export default function useCurrencyConverter(endpoint, access_key) {
+  const [currencyOptions, setCurrencyOptions] = useState([]);
+  const [fromCurrency, setFromCurrency] = useState();
+  const [toCurrency, setToCurrency] = useState();
+  const [exchangeRate, setExchangeRate] = useState();
+  const [amount, setAmount] = useState(1);
+  const [isAmountInFromCurrency, setAmountInFromCurrency] = useState(true);
+  const [currencies, setCurrencies] = useState({});
+  const [timestamp, setTimestamp] = useState();
+
+  useEffect(() => {
+    fetch(
+      `http://api.exchangeratesapi.io/v1/${endpoint}?access_key=${access_key}`
```

Comment:
Is having the url hardcoded here going to cause problems for testing? 

It would be nice to have a local copy of the returned data served, for example, with json-server for testing, and so that people won't have to create an account on exchangeratesapi.io to test the app.